src/Sylius/Bundle/AdminBundle/Resources/views/Shipment/show.html.twig

```diff
@@ -0,0 +1,35 @@
+{% extends '@SyliusAdmin/layout.html.twig' %}
+
+{% block content %}
+
+{% include '@SyliusAdmin/Shipment/Show/_header.html.twig' %}
+
+{% include '@SyliusAdmin/Shipment/Show/_breadcrumb.html.twig' %}
+
+<div class="ui segment spaceless sylius-grid-table-wrapper">
+    <table class="ui sortable stackable very basic celled table">
+        <thead>
+            <tr>
+                <th class="sylius-table-column-code">{{ 'sylius.ui.product_name'|trans }}</th>
+                <th class="sylius-table-column-code">{{ 'sylius.ui.variant_name'|trans }}</th>
+                <th class="sylius-table-column-code">{{ 'sylius.ui.quantity'|trans }}</th>
+            </tr>
+        </thead>
+        <tbody>
+            {% set currentProductName = '' %}
+            {% set currentVariantName = '' %}
+            {% for unit in shipment.units %}
+                {% if currentProductName != unit.orderItem.product.name and currentVariantName != unit.orderItem.variant.name %}
```

Comment:
What do you want to achieve here?

Reply:
I try show only unique units, and don't duplicate:
![Screenshot 2020-02-07 at 14 58 32](https://user-images.githubusercontent.com/39232096/74035432-87d74c00-49ba-11ea-9449-4827209cd97b.png)
 
Without: 
![Screenshot 2020-02-07 at 14 58 10](https://user-images.githubusercontent.com/39232096/74035459-93c30e00-49ba-11ea-9ea5-490867e25113.png)


Maybe you have any other idea?

Reply:
WDYT about some extension with a provider that will provide prepared data?

Reply:
IMHO this is to easy logic, its only iterate by collection items, I not see profits on creation new provider for this

Reply:
hmm you are setting names, checking `{% if currentProductName != unit.orderItem.product.name and currentVariantName != unit.orderItem.variant.name %}`, adding table row, setting temporary product/variant name.
It does not look like simple iteration :D